irohad/main/iroha_conf_loader.hpp

```diff
@@ -63,16 +65,38 @@ struct IrohadConfig {
   boost::optional<logger::LoggerManagerTreePtr> logger_manager;
   boost::optional<shared_model::interface::types::PeerList> initial_peers;
   boost::optional<UtilityService> utility_service;
+
+  // This is a part of cryto providers feature:
+  // https://github.com/MBoldyrev/iroha/tree/feature/hsm-utimaco.
+  // This brings unnecessary complexity, but the aim is that this config section
+  // should require no modifications from users when the feature branch is
+  // merged.
+  struct Crypto {
+    struct Default {
```

Comment:
It was not clear why this struct was called default from the first glance :( In HSM branch, it is one of the key formats, and default means Iroha default.